### PR TITLE
buvanesh2022re

### PR DESCRIPTION
**Pull request by anirudhb11**
## Project Name

`buvanesh2022re`

## Compiled PDF
[Compiled Pdf](https://github.com/anirudhb11/MLRC/blob/main/2021/buvanesh2022re/journal/article.pdf)

## Metadata Checklist

Ensure that you have added the following metadata information (put an x to the checkbox items) in `<paper_citekey>/journal/metadata.yaml`.

- [x] `title` field exists and it starts with `[Re] `.
- [x] `authors` contain author information with affiliations.
- [x] [ORCID id](https://orcid.org/) is provided for authors.
- [x] One contact author is added using `*`.
- [x] Github `url` is provided in `code` section.
- [x] [SWH identifier](https://archive.softwareheritage.org/save/) is provided in `swh`.
- [x] `replication` section is populated with the necessary information about the paper you reproduced.
- [x] `abstract` field is populated with your paper abstract (including double quotes)
- [x] `domain` should contain "ML Reproducibility Challenge 2021"
- [x] `review` section contains the url to the OpenReview forum of your paper.
- [x] Fields `contributors`, `dates`, `article` and `journal` are kept blank, as those will be populated by the Area Chairs / ReScience editors.

## Build Errors

None

### CODE REVIEW

File: 2021/buvanesh2022re/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+% ready for submission
+% \usepackage{neurips_2019}
+
+% to compile a preprint version, e.g., for submission to arXiv, add add the
+% [preprint] option:
+%     \usepackage[preprint]{neurips_2019}
+
+% to compile a camera-ready version, add the [final] option, e.g.:
+\usepackage[dvipsnames]{xcolor}
```
Comment (koustuvsinha) [inline 2021/buvanesh2022re/openreview/packages.tex:18]: Please remove this package, which is the cause of the issue (please check the README) : 

> Please do not import xcolor in your packages.tex, as rescience.cls already imports this package

Comment (anirudhb11) [reply 2021/buvanesh2022re/openreview/packages.tex:18]: @koustuvsinha removing this also doesn't solve the issue on my end. Do you have any other pointers on what the issue could be?

**Comment by koustuvsinha** [comment]
In your `openreview/appendix.tex`, the lines 403-404 are the reason for the build issue. Please comment out the lines `\bibliographystyle{plain}` and `\bibliography{supp_bibliography}`. Instead, ensure all bibliography is within `openreview/bibliography.bib` file.

Also, I found you have included xcolor twice in packages.tex. Please comment out both and fixing the above should make the build work.

File: 2021/buvanesh2022re/openreview/packages.tex (hunk 1 above)
Comment (koustuvsinha) [reply 2021/buvanesh2022re/openreview/packages.tex:18]: @anirudhb11 added a comment : comment out imports of bibliography in appendix.tex

**Comment by mdrpanwar** [comment]
> In your `openreview/appendix.tex`, the lines 403-404 are the reason for the build issue. Please comment out the lines `\bibliographystyle{plain}` and `\bibliography{supp_bibliography}`. Instead, ensure all bibliography is within `openreview/bibliography.bib` file.
> 
> Also, I found you have included xcolor twice in packages.tex. Please comment out both and fixing the above should make the build work.

Thanks, @koustuvsinha. This indeed fixed the issue.

**Comment by mdrpanwar** [comment]
@koustuvsinha Just leaving a comment to notify you that the PR is final from our end. Please consider it for merging whenever necessary.

**Comment by mdrpanwar** [comment]
@SharathRaparthy Could you please check this PR and label it appropriately (e.g. `latex ok`)? We noticed that most PRs have some label by this time, however, this PR may have been missed for some reason. Thanks!

**Comment by anirudhb11** [comment]
> @SharathRaparthy Could you please check this PR and label it appropriately (e.g. `latex ok`)? We noticed that most PRs have some label by this time, however, this PR may have been missed for some reason. Thanks!

@koustuvsinha could you please take a look into this?

**Comment by SharathRaparthy** [comment]
@anirudhb11 Yes it compiles well.  Can you please leave `Received` and `Published` that you see on the left side of the first page empty?

**Comment by anirudhb11** [comment]
> @anirudhb11 Yes it compiles well. Can you please leave `Received` and `Published` that you see on the left side of the first page empty?

@SharathRaparthy have left the `Received` and `Published` date as empty. Can we add the appropriate labels if everything looks good?

**Pull request merged**